assets/queries/dockerfile/change_default_shell/query.rego

```diff
@@ -0,0 +1,14 @@
+package Cx
+
+CxPolicy [ result ] {
+  resource := input.document[i].command[name][_]
+  resource.Cmd == "shell"
+  
+	result := {
+                "documentId": 		  input.document[i].id,
+                "searchKey": 	      sprintf("FROM={{%s}}.{{%s}}", [name, resource.Original]),
+                "issueType":		    "IncorrectValue", 
+                "keyExpectedValue": sprintf("FROM={{%s}}.{{%s}} doesn't changes the default shell", [name, resource.Original]),
+                "keyActualValue": 	sprintf("FROM={{%s}}.{{%s}} changes the default shell", [name, resource.Original]),
+              }
+}
```

Comment:
The query is not checking the vulnerability. The vulnerability is not the use of the SHELL command, is the use of the RUN command to change the default shell instead of the SHELL cmd.